frontend/js/src/user/stats/components/UserArtistActivity.tsx

```diff
@@ -166,28 +201,39 @@ export default function UserArtistActivity(props: UserArtistActivityProps) {
                     tickSize: 5,
                     tickPadding: 5,
                     tickRotation: -45,
-                    renderTick: (tick) => (
-                      <g transform={`translate(${tick.x},${tick.y})`}>
-                        {tick.value
-                          .split("\n")
-                          .map((line: string, i: number) => (
-                            <text
-                              key={line}
-                              x={0}
-                              y={10 + i * 15}
-                              textAnchor="end"
-                              dominantBaseline="middle"
-                              style={{
-                                fontSize: 10,
-                                fill: "#000",
-                                transform: `rotate(-45deg)`,
-                              }}
-                            >
-                              {line}
-                            </text>
-                          ))}
-                      </g>
-                    ),
+                    renderTick: (tick) => {
+                      const artistMbid =
+                        rawData?.result?.[tick.tickIndex]?.artist_mbid || "";
+                      const artistName =
+                        rawData?.result?.[tick.tickIndex]?.name || "";
+                      const lines = tick.value.split("\n");
+                      const linkTo = artistMbid
+                        ? `/artist/${artistMbid}`
+                        : `/search?search_term=${encodeURIComponent(
+                            artistName
+                          )}&search_type=artist`;
+                      return (
+                        <g transform={`translate(${tick.x},${tick.y})`}>
+                          <foreignObject
+                            x="-90"
+                            y="0"
+                            width="90"
+                            height="30"
+                            style={{
+                              overflow: "visible",
+                              transform: "translate(-5px, 5px) rotate(-45deg)",
+                              textAlign: "right",
+                              fontSize: "11px",
+                              lineHeight: "1.25em",
```

Comment:
I still think this should have the value of 1em, for some specific screen sizes it prevents overlapping:
![image](https://github.com/user-attachments/assets/193aa8b6-b084-4906-b6c5-8df741f32e74)
![image](https://github.com/user-attachments/assets/ddebad5f-1c5f-4b06-9a98-4efaf30670d2)